### DISCUSSION TITLE
courses/beginners-web-handbook/carousel-project-1/02-markup

### DISCUSSION THREAD

**Discussion by giscus** [crwdu/lms-contents-js101#33] [Q&A]
# courses/beginners-web-handbook/carousel-project-1/02-markup

바닐라코딩

https://www.vcdr.club/courses/beginners-web-handbook/carousel-project-1/02-markup

**Comment by jin-ttao** [comment]
안녕하세요. 간단한 질문 있습니다!

- 질문할 부분 : 
[8-2. 코드 설명] A. 기준 컨테이너가 되는 .carousel 요소 중,
"align-items를 center로 설정한 것은 자식 요소들을 수직선 상에서 가운데 정렬"로 표기되어 있습니다.

- 질문:
가로로 정렬되는 결과이다 보니, 수직선이 아닌 '수평선 상의 가운데 정렬'로 보는게 맞지 않을까 생각했습니다. 
제가 잘못 인지한 개념이 있을지 싶어 확인차 질문 드려요

**Reply by Ken123777** [reply]
안녕하세요.

제가 작성해놓은 내용이 조금 명확하지 못했나보네요. :) `.carousel` 이라는 요소의 하위 자식 요소인 방향 아이콘들을 아래 그림에서와 같이 수직선 상에서 가운데에 위치하도록 정렬하였다는 의미입니다. 아래 그림을 보시면 조금 더 도움이 될까요? (저는 수직선 상이라는 표현이 맞다고 생각했는데, 혹시 제가 틀렸다면 말씀해주세요.)

<img width="1172" alt="스크린샷 2024-07-11 오후 3 57 51" src="https://github.com/crwdu/lms-contents-js101/assets/3277615/94fa9755-986c-49dd-8b72-37470d390ad9">

**Reply by Ken123777** [reply]
주어진 상황에 따라 코드를 비활성화 했을때 다르게 반응할 수 있어서 코드 주석처리는 정확한 현상을 파악하기는 조금 무리가 있을 수 있을거 같다는 생각인데, 말씀하신 내용을 어떻게 표현하는게 더 직관적으로 이해하기 수월할지는 제가 한번 고민해볼게요. :) 

다만 지금 결국 중요한건 `align-items`를 잘 이해하시는게 중요할거 같은데, [mdn 페이지](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items)에 들어가보시면 Try it이라는 부분이 초반에 나오니 해당 부분을 한번 보시면 이해한 내용을 확인해보시기 더 좋을거 같아요 🙇🏻‍♂️

**Reply by jin-ttao** [reply]
제가 잘못 처리하고 답글을 남겨서, 삭제하고 재답글을 남겨요! 보여주신 그림으로 이해되었어요. 해당 코드의 역할을 "좌우 화살표를 내려서 이미지 수직선상 가운데 지점으로 위치시킨다"는 것으로 이해했습니다!

**Reply by Ken123777** [reply]
ㅎㅎ 넵넵! 혹시 모르니 제가 위에 남긴 mdn 페이지도 한번 살펴보세요 🥸

**Reply by jin-ttao** [reply]
헉 바로 답글을 달아주셨군요 ㅎㅎ 감사합니다 켄님
